### ISSUE TITLE
Guild treasury gives wrongs items

### ISSUE THREAD

**Issue by Cheezco** [Maselkov/GW2Bot#93]
I tried to use command to get guild treasury multiple times, but it gives me the wrong items. I'm I doing something wrong?

**Comment by Maselkov** [comment]
Post the output, and the expected output.

**Comment by Cheezco** [comment]
![unknown](https://user-images.githubusercontent.com/60855273/76024362-9f5f1180-5f33-11ea-82ce-de85f157d17b.png)
![unknown (1)](https://user-images.githubusercontent.com/60855273/76024363-9ff7a800-5f33-11ea-8256-222a47aa8010.png)
They are different within one-minute interval.

**Comment by Maselkov** [comment]
Are you sure it's not just guild members using the items up?

**Comment by Cheezco** [comment]
Yes, I'm sure. I experienced this multiple times even when no one is online
